uvm/impl/com/untangle/uvm/LocalDirectoryImpl.java

```diff
@@ -99,9 +100,19 @@ public String getRadiusLogFile()
     public String getRadiusProxyStatus()
     {
         SystemSettings systemSettings = UvmContextFactory.context().systemManager().getSettings();
-        String command = (FREERADIUS_PROXY_SCRIPT + " \"" + systemSettings.getRadiusProxyUsername() + "\" \"" + systemSettings.getRadiusProxyPassword() + "\"");
+        String userPassword;
+        try{
+            userPassword = PasswordUtil.getDecryptPassword(systemSettings.getRadiusProxyEncryptedPassword());
+            if (userPassword == null) {
+                throw new NullPointerException("Got null as decryptedPassword");
+            }
+        }catch (Exception exn) { 
+            logger.error("Exception occured while fetching original password", exn);
+            return new ExecManagerResult(1, "Unable to decrypt the encrypted password for AD server" + systemSettings.getRadiusProxyServer()).getOutput();
```

Comment:
Why create ExecManagerResult object and then return its message field ? Why not just return `"Unable to decrypt the encrypted password for AD server" + systemSettings.getRadiusProxyServer()` ?